src/Util/AttributesExtractor.php

```diff
@@ -56,12 +56,17 @@ public static function extractAttributes(array $attributes): array
         }
 
         $hasRequestAttributeKey = false;
-        foreach (OperationType::TYPES as $operationType) {
-            $attribute = "_api_{$operationType}_operation_name";
-            if (isset($attributes[$attribute])) {
-                $result["{$operationType}_operation_name"] = $attributes[$attribute];
-                $hasRequestAttributeKey = true;
-                break;
+        if (isset($attributes['_api_operation_name'])) {
```

Comment:
Can you add `TODO`s everywhere we have code blocks to remove in 3.0 please? It will ease the work a lot.